### PR TITLE
added privacy policy page

### PR DESCRIPTION
**Pull request by AswaniBolisetti**
@ajay-dhangar 
I have added a page for privacy policy
kindly check it out

https://github.com/user-attachments/assets/8a0b94da-1289-4890-a162-5d1b25a6904d

### CODE REVIEW

File: src/pages/privacy/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          <p className="text-lg text-gray-600 dark:text-gray-300">
+            You have the right to access, correct, or delete your personal information at any time. Contact us if you wish to exercise these rights.
+          </p>
+        </section>
+
+        {/* Contact Us */}
+        <section className="container mx-auto py-12 px-6 md:px-12 text-left">
+          <h2 className="text-3xl md:text-4xl font-bold text-gray-800 dark:text-white mb-4">6. Contact Us</h2>
+          <p className="text-lg text-gray-600 dark:text-gray-300">
+            If you have any questions or concerns regarding this Privacy Policy, feel free to contact us at <a href="mailto:support@algowebsite.com" className="text-blue-600 dark:text-yellow-400">support@algowebsite.com</a>.
```
Comment (ajay-dhangar) [inline src/pages/privacy/index.tsx:88]: replace `support@algowebsite.com` with `ajaydhangar49@gmail.com `

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
change email based on suggestions

**Comment by AswaniBolisetti** [comment]
@ajay-dhangar 
Could you please confirm whether email changed or not

**Comment by ajay-dhangar** [comment]
> @ajay-dhangar Could you please confirm whether email changed or not

You need to push the code to this branch. Once the code is moved, I can then merge the changes.

**Comment by AswaniBolisetti** [comment]
@ajay-dhangar 
Kindly review and merge it 
Please

**Comment by AswaniBolisetti** [comment]
@ajay-dhangar 
Could you please tell me why it was closed

**Pull request closed without merging**